app/views/posts/index.html.erb

```diff
@@ -27,11 +27,8 @@
         </h3>
```

Comment:
| Your User Show | Wireframe |
| --- | --- |
| ![image](https://github.com/anas599/blog/assets/93039359/1ce270ae-0dac-4086-bd35-dda527193185) | ![user index](https://raw.githubusercontent.com/microverseinc/curriculum-rails/main/blog-app/images/blog_user_page.png?token=GHSAT0AAAAAACAAABAFX5SV6RPIFMVZWOGSZC2MN3A) |

| Your Post Index | Wireframe |
| --- | --- |
| ![image](https://github.com/anas599/blog/assets/93039359/549ddd59-6a83-437a-9d55-439e9ff7c8ea) | ![user index](https://raw.githubusercontent.com/microverseinc/curriculum-rails/main/blog-app/images/blog_user_page.png?token=GHSAT0AAAAAACAAABAFX5SV6RPIFMVZWOGSZC2MN3A) |


| Your Post Show | Wireframe |
| --- | --- |
| ![image](https://github.com/anas599/blog/assets/93039359/f2ccd465-d583-4407-bf94-6447a919afbf) | ![user index](https://raw.githubusercontent.com/microverseinc/curriculum-rails/main/blog-app/images/blog_user_page.png?token=GHSAT0AAAAAACAAABAFX5SV6RPIFMVZWOGSZC2MN3A) |

- [ ] Great work on making your app work 👍 but I guess while making the views you forgot to add the comments counter and likes counter 😨 Kindly add them in all the above views to make them follow the wireframe 😊
- [ ] It would be great if you fix the margins, paddings and alignments of your project views to make them follow the wireframe to make them attractive 😊